redocly.yaml

```diff
@@ -0,0 +1,23 @@
+# See https://redocly.com/docs/cli/configuration/
```

Comment:
The Redocly extension was prompting if we wanted to create a config file. Added it so that everyone doesn't get annoyed by the prompts.